gramex/handlers/mlhandler.py

```diff
@@ -112,14 +114,26 @@ def setup(cls, data=None, model={}, config_dir='', **kwargs):
             data = cls._filtercols(data)
             data = cls._filterrows(data)
             cls.model = cls._assemble_pipeline(data, mclass=mclass, params=params)
-
             # train the model
             target = data[target_col]
             train = data[[c for c in data if c != target_col]]
+            # cross validation
+            mod = cls.modelFunction()
```

Comment:
This is not required. The model is already present as `cls.model`, see line no: 116.